worker/task.go

```diff
@@ -781,6 +798,17 @@ func parseSrcFn(q *protos.Query) (*functionContext, error) {
 		}
 		checkRoot(q, fc)
 		fc.lang = q.SrcFunc[1]
+	case UidInFn:
+		if err = ensureArgsCount(q.SrcFunc, 1); err != nil {
+			return nil, err
+		}
+		if fc.uidPresent, err = strconv.ParseUint(q.SrcFunc[2], 10, 64); err != nil {
+			return nil, err
+		}
+		checkRoot(q, fc)
+		if fc.isFuncAtRoot {
+			return nil, x.Errorf("uid_in function not allowed at root")
```

Comment:
@ashwin95r @manishrjain is there any reason that `uid_in` is not supported as a root func? This would be super helpful! Happy to help if possible.